.github/workflows/biothings_annotator_tests.yml

```diff
@@ -0,0 +1,43 @@
+name: Biothings Annotator Tests
+
+on:
+  push: {}
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  run_user_geneset_tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [ "3.9", "3.10", "3.11", "3.12" ]
+    steps:
+      - name: Checkout biothings_annotator source
+        uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Upgrade pip
+        run: pip install --upgrade pip
+      - name: Create Python Virtual Environment
+        run: python -m venv .venv
+      - name: Source Python Virtual Environment
+        run: source .venv/bin/activate
+      - name: Install PyTest
+        run: pip install pytest
+      - name: Install Sanic
+        run: pip install sanic
+      - name: Install biothings_client
+        run: pip install biothings_client
+      - name: Install brotli
+        run: pip install brotli
+      - name: Install sanic_testing
+        run: pip install sanic_testing
+      - name: Run Tests - Curie
+        run: python -m pytest tests/test_curie.py
```

Comment:
Do you want to run the entire test suite? This way we get a little more test coverage when we create our PR's

Reply:
yes, we can run all tests.

Reply:
There are few tests failing, after changing to run all tests from the `tests/` directory.